server.js

```diff
@@ -95,7 +104,16 @@ app.get('/auth', (req, res, next) => {
 
 // This routing path handles the authentication callback.
 // This path (including the host information) must be configured in Banno SSO settings.
-app.get('/auth/cb', (req, res, next) =>
+app.get('/auth/cb', (req, res, next) => {
+  // This line is meant as a workaround for a specific issue that presents itself when attempting to use
+  // this example project to build a plugin for the Plugin Framework.
+  //
+  // This works around a quirk in how sessions are handled by this project's
+  // OpenID Connect client (https://github.com/panva/node-openid-client) dependency.
+  //
+  // This line is NOT recommended for use in production systems.
```

Comment:
I'd simply say that this is an undocumented workaround for this particular OAuth2 client and that the implementer must ensure that protections must be put in place to ensure that a request arriving without an existing session and state don't create cross-site request forgery vulnerabilities.

Reply:
Addressed (hopefully) in https://github.com/Banno/consumer-api-openid-connect-example/pull/17/commits/2b57fda5b2cec92d572f669d6164d0c3bd361ae0